SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Utils/ErrorLogHelper.cs

```diff
@@ -193,6 +193,29 @@ public static void RemoveStoredErrorLogFile(Guid errorId)
             }
         }
 
+        /// <summary>
+        /// Removes all stored error log files.
+        /// </summary>
+        public static void RemoveAllStoredErrorLogFiles()
+        {
+            lock (LockObject)
+            {
+                foreach (var file in GetErrorLogFiles())
```

Comment:
This can be simplified to `CrashesDirectory.Delete(true)`, which recursively deletes the directory where crashes are stored.